app/models/compliance.rb

```diff
@@ -66,13 +66,10 @@ def self.check_compliance(target, _inputs = {})
                                                                                          :number     => id}
       end
     end
-    target_class = target.class.base_model.name.downcase
-    target_class = "vm" if target_class.match("template")
```

Comment:
do we want to use `compliance_policy_name` here?
That fixes the template => vm swap
but it will also change physical server and container images change.

Which I do not know if we want to deal with this change.

Reply:
I think that makes sense, yes...Might as well put the "override" where it belongs.

Reply:
so it looks like there is only one override (vm and template) which is totally valid.
The others were unnecessarily overridden. (the difference between `underscore` and `downcase`)